<!-- 
Instructions on how to make a launch file
pkg = package name. It has to be the same of the CMakeLists.txt
type = name of pyhton file that you want to execute
name = name of the node  (rospy.init_node('my_algorithm_node', anonymous=True) 
-->

<launch>
    <node pkg="qr_detector_pkg" type="qr_detector_cam.py" name="qr_detector_node_cam" output="screen">
     <param name="camera_topic_name" value="/usb_cam/image_raw" />
     <param name="camera_info_topic_name" value="/usb_cam/camera_info" />
     <param name="output_topic_name" value="/output" />
    </node>
    <!--<node name="image_view_result" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/output"/>
        <param name="autosize" value="true" />
    </node>-->

</launch>
